_data/icons/rinia.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://QmX7Lhq9om6kYN9phrWstHd81i6wpNPTXBetw2UgTFpjVJ",
+    "width": 512,
+    "height": 512,
```

Comment:
size is wrong:

```
 identify QmX7Lhq9om6kYN9phrWstHd81i6wpNPTXBetw2UgTFpjVJ         23:00:05
QmX7Lhq9om6kYN9phrWstHd81i6wpNPTXBetw2UgTFpjVJ PNG 2000x2000 2000x2000+0+0 8-bit sRGB 653045B 0.000u 0:00.000

```

Reply:
oke, thank you sir. Making changes to it now. Thank you